scene/2d/path_2d.cpp

```diff
@@ -232,9 +232,8 @@ void PathFollow2D::_update_transform() {
 
 	if (rotates) {
 		Transform2D xform = c->sample_baked_with_rotation(progress, cubic);
-		xform.translate_local(v_offset, h_offset);
-		set_rotation(xform[1].angle());
-		set_position(xform[2]);
+		xform.translate_local(h_offset, v_offset);
+		set_transform(xform);
```

Comment:
This makes it impossible to scale `PathFollow2D` which was previously possible

Reply:
Thanks for the insights! I was in a rush at that moment, but I think my fix now is cleaner.